data/json/npcs/isolated_road/isolated_road_cody_dialogue.json

```diff
@@ -70,7 +69,7 @@
         "topic": "TALK_BLACKSMITH_PROTOTYPE_EXODII"
       },
       {
-        "text": "[100 merch] I'd like to become a member of the co-op.",
+        "text": "[100 Merch] I'd like to become a member of the co-op.",
```

Comment:
Yeah, I was honestly thinking about that too. It's historically been capitalized, such as in the item name and in other dialogue that wasn't written by me. It might behoove me to check in with a dev and then leave a comment or something on the item def.

Reply:
so i got in touch with i-am-erk and it looks like we want to avoid capitalizing currency like this. a corner case could be "Hub 01 coin", but the currency name there isn't really the whole thing, it's a coin which is Hub 01's. we might want a more distinct name (or at least slang) for the hub currency; he suggested 'hubcoin' or 'hubcap', which i think is hilarious, but those details are out of scope for now.

point is: we want to use 'merch' for *this* currency.

Reply:
perfect, okay! I'm real happy to finally have a guideline for that. I'll lowercase it sometime soon, maybe after I play D&D tonight.